src/Sulu/Bundle/AdminBundle/Admin/Routing/DropdownToolbarAction.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\AdminBundle\Admin\Routing;
+
+class DropdownToolbarAction extends ToolbarAction
+{
+    public function __construct(string $label, string $icon, array $actions)
```

Comment:
Arrays should have a phpdoc typehint with accepted types for static type checking:

```
/** 
 * @param ToolbarAction[] $actions
 */
public function __construct(string $label, string $icon, array $actions)
```

Reply:
I personally would also name the variable $toolbarActions to be as specific as possible but think we should then also rename the option parameter in the js component.